crates/nargo/src/cli/fs/program.rs

```diff
@@ -27,7 +27,7 @@ pub(crate) fn save_acir_hash_to_dir<P: AsRef<Path>>(
     hash_dir: P,
 ) -> PathBuf {
     let acir_hash = hash_constraint_system(circuit);
-    let hash_path = hash_dir.as_ref().join(hash_name).with_extension("json.sha256");
+    let hash_path = hash_dir.as_ref().join(hash_name).with_extension("json.checksum");
```

Comment:
> I think that it'd be nice to have a constant for this extension in order to avoid the same problem in the future.

Agreed, I should have done this. Could you add `ACIR_HASH` to `constants.rs`?

Reply:
Sure!